src/components/ShoppingListItem.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import * as timeframeConstants from '../lib/timeframeConstants';
+
+const timeFrameString = number =>
+{

```

Comment:
This is not a blocker at all, but maybe these 2 functions could be combined into one since they both do essentially the same thing? The case could return an object that has both the css class and the label in it? idk.

Reply:
Yes, I was thinking the same thing. I was thinking of dropping the values in 2 dictionaries with the numbers as the keys, and having a 2nd argument to determine what you want returned.